chia/rpc/wallet_rpc_api.py

```diff
@@ -1487,12 +1487,31 @@ async def cat_spend(self, request, hold_lock=True) -> EndpointResult:
             }
         else:
             exclude_coins = None
+        cat_discrepancy_params: Tuple[Optional[int], Optional[str], Optional[str]] = (
```

Comment:
why "discrepancy"?

Reply:
"discrepancy in accounting" is how we describe extra delta and the related calculations that need to be adjusted while that CATs are making their accounting announcements